lonestar/analytics/distributed/README.md

```diff
@@ -4,46 +4,42 @@ Overview of Distributed and Heterogeneous Systems in Galois
 This directory contains benchmarks that run using D-Galois and D-IrGL.
 
 D-Galois is distributed Galois built using the Gluon communication substrate.
-Similarly, D-IrGL is distributed IrGL built using Gluon. 
+Similarly, D-IrGL is distributed IrGL built using Gluon.
 Gluon is just the communication substrate: it is not a standalone system.
 
 Basic Compiling Through CMake (Distributed and Heterogeneous Galois)
 ================================================================================
 
 The dependencies for distributed Galois are exactly the same as shared-memory
-Galois except that it requires an MPI library (e.g. mpich2) to be on the 
+Galois except that it requires an MPI library (e.g. mpich2) to be on the
 system as well.
 
-To build distributed/heterogeneous Galois, certain CMake flags must be 
+To build distributed/heterogeneous Galois, certain CMake flags must be
 specified.
 
 For distributed Galois, i.e. D-Galois:
 
 `cmake ${GALOIS_ROOT} -DGALOIS_ENABLE_DIST=1`
 
-For heterogeneous (which is also distributed) Galois, i.e. D-IrGL:
+For heterogeneous Galois, i.e. D-IrGL:
```

Comment:
This should be "For distributed and heterogeneous Galois". 

ENABLE_GPU does not enable heterogeneous galois, it also enables lonestar GPU. ENABLE_GPU with ENABLE_DIST enables heterogeneous Galois.

Reply:
Good catch; relic of before the flag updates. Will fix.